addons/edi/models/edi_document.py

```diff
@@ -99,7 +99,7 @@ def _default_project_id(self):
     _sql_constraints = [("model_uniq", "unique (model_id)", "The document model must be unique")]
 
     @api.model
-    def autocreate(self, inputs):
+    def autocreate(self, inputs, allow_process=True):
```

Comment:
Nit: may be name the parameter as `auto_process`? (Feel free to ignore if you prefer the existing name).

Reply:
The parameter name is same with variable name on gateway class i think